packages/app-project/stores/UI.js

```diff
@@ -6,7 +6,7 @@ const UI = types
   .model('UI', {
     mode: types.optional(types.enumeration('mode', ['light', 'dark']), () => {
       const mode = getCookie('mode')
-      return mode || 'light'
+      return (mode) ? mode : 'light'
```

Comment:
PFE prefers `||` to set defaults and warns agains using ternary expressions like this. Do we want to carry that over into the new app?
http://eslint.org/docs/rules/no-unneeded-ternary.html

Reply:
FWIW, I find `||` a bit harder to read for some reason and I changed it when I first noticed that sometimes `getCookie` was returning an empty string. I meant to have switched it back.

I'm still of the opinion that if we want to match PFE's linting rules, then we should drop standard and switch back to the airbnb rule set then. I don't have a strong opinion about which rule set we follow.